dvc/command/status.py

```diff
@@ -61,6 +66,8 @@ def run(self):
                 logger.info(json.dumps(st))
             elif st:
                 self._show(st, indent)
+            elif not self.repo.stages:
+                logger.info(self.EMPTY_PROJECT_MSG)
             else:
                 logger.info(self.UP_TO_DATE_MSG)
```

Comment:
Nice, thanks @DDR0 !

We should check whether this affects any command line sample blocks in docs... ⌛

Reply:
There's 18 instances of the `Data and pipelines are up to date.` text [in our docs repo](https://github.com/iterative/dvc.org/search?q=Data+and+pipelines+are+up+to+date.), some of these examples may now be the new text, not sure. I've unmarked the docs check box for now.

This is minor BTW, not big deal.

Reply:
OK never mind. I checked all the instances and none seem to be for this case (no data/pipelines in the project yet). All good here!